scc/src/opts.rs

```diff
@@ -0,0 +1,147 @@
+use bpaf::*;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::collections::VecDeque;
+
+#[cfg(test)]
+use std::{println as info};
```

Comment:
this import can be a bit misleading since there's also `log::info`, can we just call `println` directly?